### ISSUE TITLE
TypeError: chrono_Chrono is not a constructor

### ISSUE THREAD

**Issue by rayepps** [wanasit/chrono#331]
I get this error in my node v14 module and in the repl when I import chrono. So error occurs as soon as I run the api. I just installed chrono to my project that has been working doing some misc nlp things. Today ran `yarn add chrono-node` then added the import statement, then tried to run the api and... 💣 💥 
```
TypeError: chrono_Chrono is not a constructor
    at Module.<anonymous> (/Users/rayepps/projects/kudos/api/cloud-run-services/node_modules/chrono-node/dist/chrono.js:1454:14)
    at __webpack_require__ (/Users/rayepps/projects/kudos/api/cloud-run-services/node_modules/chrono-node/dist/chrono.js:30:30)
    at /Users/rayepps/projects/kudos/api/cloud-run-services/node_modules/chrono-node/dist/chrono.js:94:18
    at /Users/rayepps/projects/kudos/api/cloud-run-services/node_modules/chrono-node/dist/chrono.js:97:10
    at webpackUniversalModuleDefinition (/Users/rayepps/projects/kudos/api/cloud-run-services/node_modules/chrono-node/dist/chrono.js:3:20)
    at Object.<anonymous> (/Users/rayepps/projects/kudos/api/cloud-run-services/node_modules/chrono-node/dist/chrono.js:10:3)
    at Module._compile (internal/modules/cjs/loader.js:1200:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
    at Module.load (internal/modules/cjs/loader.js:1049:32)
    at Function.Module._load (internal/modules/cjs/loader.js:937:14)
```

### In the REPL
```
> const chrono = require('chrono-node')
```

### In my esmodule
```
import * as chrono from 'chrono-node'
```

### Environment
This is on a node api project running node `v14.3.0`. Installed chrono in package.json is `"chrono-node": "^2.0.0"`. yarn.lock says version is actually `2.0.0` (expected).

**Comment by rayepps** [comment]
Forcing the version to `1.4.6` (latest v1 version as far as I can tell) made for a workaround.

**Comment by wanasit** [comment]
thanks for reporting this. I apologize for the problem. This should be fixed in `2.0.2+`

**Comment by rayepps** [comment]
I upgraded to version `2.0.2` and it works as expected. @wanasit thanks for the quick response! You rock 🥇
